cmake/SetupCompilerOptions.cmake

```diff
@@ -229,8 +229,18 @@ if( BLT_CXX_STD STREQUAL c++98 )
     set(CMAKE_CXX_STANDARD 98)
 elseif( BLT_CXX_STD STREQUAL c++11 )
     set(CMAKE_CXX_STANDARD 11)
+    if (COMPILER_FAMILY_IS_XL)
```

Comment:
I am running into this issue with Intel also. CMAKE_CXX_STANDARD does not add the flags. I've not checked this with 3.8.x yet